ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/avdtestbase.py

```diff
@@ -4,31 +4,34 @@
 from __future__ import annotations
 
 import logging
+from collections.abc import Mapping
 from functools import cached_property
 from ipaddress import ip_interface
 
 from ansible_collections.arista.avd.plugins.plugin_utils.errors import AristaAvdError, AristaAvdMissingVariableError
 from ansible_collections.arista.avd.plugins.plugin_utils.utils import get, get_item
 
 LOGGER = logging.getLogger(__name__)
+LOGGING_LEVELS = ["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"]
 
 
 class AvdTestBase:
     """
     Base class for all AVD eos_validate_state tests.
     """
 
-    def __init__(self, device_name: str, hostvars: dict):
+    def __init__(self, device_name: str, hostvars: dict | object):
         """
         Initialize the AvdTestBase class.
 
         Args:
             device_name (str): The current device name for which the plugin is being run.
-            hostvars (dict): A dictionary that contains a key for each device with a value of the structured_config.
-                           When using Ansible, this is the equivalent of `task_vars['hostvars']`.
+            hostvars (dict | object): A dictionary that contains a key for each device with a value of the structured_config.
+                                      When using Ansible, this is the `task_vars['hostvars']` object.
         """
         self.hostvars = hostvars
         self.device_name = device_name
+        self.struct_cfg = self.get_host_struct_cfg(host=device_name)
```

Comment:
I would probably call this `self.structured_config` here to be clear though.

Reply:
Fixed in 22836aff3e4102ab512a54b400ec10a4213cff27

Reply:
I don't see any change for this.